android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/ble/AdvertiseDataUtils.java

```diff
@@ -0,0 +1,203 @@
+/*
+Helper class to simplify Eddystone URL encoding
```

Comment:
Add the normal copyright header.  Move this comment just above the class definition.